metrics/web/test/common/presentation/navigation/route_configuration/route_configuration_test.dart

```diff
@@ -1,76 +1,254 @@
 // Use of this source code is governed by the Apache License, Version 2.0
 // that can be found in the LICENSE file.
 
+import 'package:flutter/widgets.dart';
 import 'package:metrics/common/presentation/navigation/route_configuration/route_configuration.dart';
 import 'package:metrics/common/presentation/navigation/route_configuration/route_name.dart';
 import 'package:test/test.dart';
 
-import '../../../../test_utils/matchers.dart';
-
 void main() {
   group("RouteConfiguration", () {
-    const name = RouteName.login;
-    const path = 'test_path';
-    const authorizationRequired = false;
     const parameters = {'test': 'test'};
 
     test(
-      "throws an AssertionError if the given name is null",
+      ".loading() creates an instance with the the correct name",
```

Comment:
🏷️ 
Let's specify at least the correct name and path for routes in descriptions.
```suggestion
      ".loading() creates an instance with the loading route name",
```